language/02-guidelines/01-style/accessibility/index.md

```diff
@@ -32,6 +44,8 @@ When choosing colors and contrast ratios for typography it is important to adher
 
 Colors at a 50 value or lower are not accessible on neutral backgrounds, unless they're displayed at 18 pt or larger. There are no formal color contrast requirements for icons or glyphs that are paired with supporting text. If an icon is used separately from supporting text on a neutral background, use a color value of 50 or higher.
```

Comment:
Should we say "24 px" instead of "18 pt"  since that is mostly what our designers used? I think pt is more common for print design. Just to eliminate all confusion. I've made this mistake before myself. :-(